src/VirtualClient/VirtualClient.Main/profiles/PERF-KAFKA.json

```diff
@@ -0,0 +1,249 @@
+        "Role": "Server"
+      }
+    },
+    {
+      "Type": "KafkaClientExecutor",
+      "Parameters": {
+        "Scenario": "Setup-topic-replication-factor-[1]-sync",
+        "PackageName": "kafka",
+        "Port": "$.Parameters.ServerPort",
+        "WarmUp": true,
+        "CommandLine": "--create --topic sync-test-rep-one --partitions 6 --replication-factor 1 --bootstrap-server {0}:{Port}",
+        "Role": "Client",
+        "CommandType": "Setup"
+      }
+    },
+    {
+      "Type": "KafkaClientExecutor",
+      "Parameters": {
+        "Scenario": "Setup-topic-replication-factor-[ServeInstances]-sync",
+        "PackageName": "kafka",
+        "Port": "$.Parameters.ServerPort",
+        "WarmUp": true,
+        "ServerInstances": "$.Parameters.ServerInstances",
+        "CommandLine": "--create --topic sync-test-rep-{ServerInstances} --partitions 6 --replication-factor {ServerInstances} --bootstrap-server {0}:{Port}",
+        "Role": "Client",
+        "CommandType": "Setup"
+      }
+    },
+    {
+      "Type": "KafkaClientExecutor",
+      "Parameters": {
+        "Scenario": "Producer-batchSize-8196-sync",
+        "PackageName": "kafka",
+        "Port": "$.Parameters.ServerPort",
+        "WarmUp": true,
+        "ServerInstances": "$.Parameters.ServerInstances",
+        "RecordSize": "$.Parameters.RecordSize",
+        "NumberOfRecords": "$.Parameters.NumberOfRecords",
+        "CommandLine": "--topic sync-test-rep-one --num-records {NumberOfRecords} --record-size {RecordSize} --throughput -1 --producer-props acks=1 bootstrap.servers={0}:{Port} buffer.memory=67108864 batch.size=8196",
+        "Role": "Client",
+        "CommandType": "ProducerTest"
+      }
+    },
+    {
+      "Type": "KafkaClientExecutor",
+      "Parameters": {
+        "Scenario": "Producer-batchSize-64000-sync",
+        "PackageName": "kafka",
+        "Port": "$.Parameters.ServerPort",
+        "WarmUp": true,
+        "ServerInstances": "$.Parameters.ServerInstances",
+        "RecordSize": "$.Parameters.RecordSize",
+        "NumberOfRecords": "$.Parameters.NumberOfRecords",
+        "CommandLine": "--topic sync-test-rep-{ServerInstances} --num-records {NumberOfRecords} --record-size {RecordSize} --throughput -1 --producer-props acks=1 bootstrap.servers={0}:{Port} buffer.memory=67108864 batch.size=64000",
+        "Role": "Client",
+        "CommandType": "ProducerTest"
+      }
+    },
+    {
+      "Type": "KafkaClientExecutor",
+      "Parameters": {
+        "Scenario": "Consumer-sync",
+        "PackageName": "kafka",
+        "Port": "$.Parameters.ServerPort",
+        "ServerInstances": "$.Parameters.ServerInstances",
+        "NumberOfRecords": "$.Parameters.NumberOfRecords",
+        "CommandLine": "--topic sync-test-rep-one --messages {NumberOfRecords} --bootstrap-server {0}:{Port}",
+        "Role": "Client",
+        "CommandType": "ConsumerTest"
+      }
+    },
+    {
+      "Type": "ParallelExecution",
```

Comment:
Need to change from Parallel Execution to running multiple processes within client executor.

Reply:
PR is updated where we no longer use ParallelExecution, instead we will use ClientInstances as a parameter.